src/main/java/com/Acrobot/ChestShop/Commands/AccessToggle.java

```diff
@@ -0,0 +1,40 @@
+package com.Acrobot.ChestShop.Commands;
+
+import com.Acrobot.ChestShop.Listeners.Player.PlayerInteract;
+import com.Acrobot.ChestShop.Permission;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+/**
+ * @author g--o
+ *
+ * Command for toggling chest access
+ */
+public class AccessToggle implements CommandExecutor {
+    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+        if (cmd.getName().equals("csaccess")) {
+            String resultMsg = "Access to shops is ";
```

Comment:
The messages from this command will need to follow the same format as other commands (Look at `/cstoggle` for reference)

Reply:
I agree, messages should be all configurable via the message config file.